### ISSUE TITLE
Nodemon restarting at every webpack build.

### ISSUE THREAD

**Issue by BenderV** [gurunate/nodemon-browsersync-webpack-plugin#1]
It's seems that the `compile` event is triggered every time webpack make a compilation. So, even if I'm only modifying a front-end code, the nodemon process is restarted and the browser-sync is broken. 

Do you have the same issue?

**Comment by gurunate** [comment]
I've seen a similar issue.  I was able to fix it for my use by having Nodemon ignore both the source and distribution directories.  In my `nodemon.json` or the nodemon configuration in the `webpack.config.js` file I added the following settings:

```javascript
"ignore": [
    ".git",
    ".eslintrc",
    "node_modules/**/node_modules",
    "src/**",
    "public/**",
    "webpack.config.js"
  ],
```

with `src` being my source directory and `public` where **webpack** is configured to _output_ it's bundle.

Hope this helps.
